test/spec/change-streams/unified/change-streams-disambiguatedPaths.yml

```diff
@@ -15,6 +15,7 @@ createEntities:
 
 runOnRequirements:
   - minServerVersion: "6.1.0"
+    maxServerVersion: "8.1.0"
```

Comment:
With https://github.com/mongodb/node-mongodb-native/pull/4495 merged I think we can remove these 2 changes now.